### PR TITLE
Add accessibility statement link to footer

### PR DESCRIPTION
**Pull request by domoscargin**
https://trello.com/c/rRLFjASj/240-publish-accessibility-statement-by-wednesday

This adds a link to the accessibility statement on the footer component

### CODE REVIEW

File: src/digitalmarketplace/components/footer/__snapshots__/template.test.js.snap [hunk 1]
```diff
@@ -95,6 +95,11 @@ exports[`footer renders a footer component with all our standard links 1`] = `
                     Privacy notice
                   </a>
                 </li>
+                <li class=\\"govuk-footer__inline-list-item\\">
+                  <a class=\\"govuk-footer__link\\" href=\\"http://www.github.com\\">
```
Comment (gidsg) [inline src/digitalmarketplace/components/footer/__snapshots__/template.test.js.snap:99]: Needs updating to live URL!

**Review by gidsg — CHANGES_REQUESTED** [review]
We need to update this to the live URL!

File: src/digitalmarketplace/components/footer/template.test.js [hunk 2]
```diff
@@ -19,6 +19,8 @@ const urlForMock = jest.fn((param) => {
     value = 'http://www.amazon.co.uk'
   } else if (param === 'external.terms_and_conditions') {
     value = 'http://www.yahoo.co.uk'
+  } else if (param === 'external.accessibility_statement') {
+    value = 'http://www.github.com'
```
Comment (bjgill) [inline src/digitalmarketplace/components/footer/template.test.js:23]: Nit - should we be using http://example.com for this? It seems a bit odd to be putting github in our tests...

**Review by bjgill — APPROVED** [review]
Looks sensible. Just one non-blocking suggestion

File: src/digitalmarketplace/components/footer/__snapshots__/template.test.js.snap (hunk 1 above)
Comment (domoscargin) [reply src/digitalmarketplace/components/footer/__snapshots__/template.test.js.snap:99]: I think that might be out of scope for this - this testing snapshot looks like it's using dummy URLs for all of these footer links.

File: src/digitalmarketplace/components/footer/template.test.js (hunk 2 above)
Comment (domoscargin) [reply src/digitalmarketplace/components/footer/template.test.js:23]: Totally fair - was following the previous test cases, but I'll just change them all.

**Review by gidsg — APPROVED** [review]

**Pull request merged**